client-server/src/main/java/JsonStrategy.java

```diff
@@ -0,0 +1,57 @@
+import com.google.gson.*;
+
+import java.io.*;
+import java.lang.reflect.Type;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+public class JsonStrategy implements MessageStrategy {
+
+    private Gson GSON;
```

Comment:
Why `GSON` has all capital letters? Is it constant?

Reply:
no, sorry, fixed it
I probably considered it a very important object and tried to express my respect for it))